server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -243,7 +243,15 @@ cics_check_activity: (ACQPROCESS | ACTIVITY cics_data_value | ACQACTIVITY | (ABC
                      (MODE | SUSPSTATUS | COMPSTATUS) cics_cvda | cics_handle_response)+;
 cics_check_timer: (TIMER cics_data_value | STATUS cics_cvda | cics_handle_response)*;
 
-cics_conditions: EOC | EODS | INVMPSZ | INVPARTN | INVREQ | MAPFAIL | PARTNFAIL | RDATT | UNEXPIN | ERROR | DUPREC;
+cics_conditions: NORMAL | ERROR | RDATT | WRBRK | EODS | EOC | INBFMH | ENDINPT | NONVAL | NOSTART | TERMIDERR | FILENOTFOUND | NOTFND | DUPREC | DUPKEY |
```

Comment:
There seems to be one more condition `EOF` - the token will have to be named differently e.g. EOF_TOKEN.

Reply:
https://www.ibm.com/docs/en/cics-ts/6.x?topic=reference-eib-fields#dfhp4au__title__27
```
04 EOF
```